src/basic-components.md

```diff
@@ -21,11 +21,14 @@ You have heard about Biotope. Biotope is a Component Ecosystem we develop in the
 
```

Comment:
I think it would be cool to recommend the biotope generator here. 
Therefore the way where things are located is depending on the configuration. If it is a small component the template is included in the [component-name].js. Only for complex components the template is a seperate file

Reply:
like this? :)